### DISCUSSION TITLE
无法使用百度云盘

### DISCUSSION THREAD

**Discussion by xhemj** [AlistGo/alist#495] [Q&A] [answered]
按照要求配置后昨天还可以正常使用，今天一进就报错了。
报错内容：errno: -7, refer to https://pan.baidu.com/union/doc/

![](https://s4.ax1x.com/2022/02/06/HKK7B8.jpg)

**Comment by xhofe** [comment, marked as answer]
文件或目录无权访问。

**Reply by xhemj** [reply]
是指百度网盘中的目录无权访问是吗？

**Reply by xhemj** [reply]
重新创建文件夹后好像可以了，谢谢

**Reply by xhofe** [reply]
errno是百度的api返回的，这个接口https://pan.baidu.com/union/doc/nksg0sat9 ，-7它只说了文件或目录无权访问，具体我也不清楚。
